src/plone/restapi/deserializer/dxfields.py

```diff
@@ -46,8 +46,7 @@ def __call__(self, value):
         try:
             # Parse ISO 8601 string with Zope's DateTime module
             # and convert to a timezone naive datetime in local time
-            value = DateTime(value).toZone(DateTime().localZone()).asdatetime(
-            ).replace(tzinfo=None)
+            value = DateTime(value).toZone('UTC').asdatetime()
```

Comment:
@tisto @buchi @lukasgraf 

I just tried a new approach and change the code to store the datetime objects as datetime offset-aware and referenced always to UTC. It felt bad to convert it to the local time of the server each time, and it caused a bad offset when the API waas used by people in other timezones than the server's one.

Reply:
That is perfect, It should always be in an isoformat with UTC timezone.